backend/pkg/server/server.go

```diff
@@ -132,6 +134,20 @@ func New(conf *config.Config, db *db.API) (*echo.Echo, error) {
 		e.DefaultHTTPErrorHandler(err, c)
 	}
 	codegen.RegisterHandlers(e, handlers)
+
+	// setup background job for updating instance stats
+	go func() {
+		ticker := time.NewTicker(db.GetDefaultInterval())
```

Comment:
```suggestion
		err := db.UpdateInstanceStats(nil, nil)
			if err != nil {
				logger.Err(err).Msg("Error updating instance stats")
			}
		ticker := time.NewTicker(db.GetDefaultInterval())
```
It would be good to do a run on startup. The ticker will not directly issue an event.

Reply:
Maybe we can add a check that this is only done initially if there is no "recent" entry.

Reply:
By the way, I wondered why it didn't work with a local instance registering even after adding this initial run and it turned out that the default `time.Now` is not UTC. I've filed a PR: https://github.com/flatcar/nebraska/pull/664

Reply:
> Maybe we can add a check that this is only done initially if there is no "recent" entry.

On the other hand it's nice for testing to directly see something after a restart. I guess it's ok to always run it once on start.

Reply:
I've been making changes locally to test, is there a formal way to test that this works as intended (i.e. proper entries, no duplicates, etc.)?

Reply:
A test case is possible, but I think it's ok how it is now.